CONTRIBUTING.md

```diff
@@ -138,6 +141,34 @@ Rebasing is one way to integrate changes from one branch onto another, in this c
 - don't rebase, but instead `git checkout topic/branch-description`, then `git pull`, followed by `git push origin`
 	- The result will be that your pull request will include your commits in addition to all other commits to `develop` since your branch was created (without changing the commit history).
 
+## Generating a crash report
+
+### macOS
+
+Crash logs are in `~/Library/Logs/DiagnosticReports`. The following command will list the crash reports for a SuperCollider program with the most recent first:
+
+    ls -lt ~/Library/Logs/DiagnosticReports | grep -E 'SuperCollider|sclang|scsynth|supernova'
+
+If the app is hanging, and you think you know which one it is, you can force it to crash and produce a log by sending it a segfault signal:
+
+    pkill -SIGSEGV <sc-executable> # may need to execute twice to force a crash
+
+The crash log will be placed in `~/Library/Logs/DiagnosticReports` with the others.
+
+### Linux
+
+A core dump file is generated when an application crashes. See [this helpful article](https://linux-audit.com/understand-and-configure-core-dumps-work-on-linux/) for information on core dumps and how to enable them on your machine. You don't need to send us the full core dump (it will probably be quite large), but you can generate a helpful backgrace with gdb:
+
+    gdb <sc-executable> <core-file> -ex where -ex exit
```

Comment:
Are you certain that `exit` is correct here? Did you mean to write `-ex quit`?

Reply:
yep, thanks for catching that!